public/docs/ts/latest/guide/structural-directives.jade

```diff
@@ -574,9 +599,10 @@ a#summary
 
 :marked
   You learned
+
   * that structural directives manipulate HTML layout.
   * to use [`<ng-container>`](#ngcontainer) as a grouping element when there is no suitable host element.
-  * that the angular "de-sugars" [asterisk (\*) syntax](#asterisk) into a `<template>`.
+  * that the angular "de-sugars" [asterisk (*) syntax](#asterisk) into a `<template>`.
```

Comment:
Done.
Also, I dropped the hyphen in "desugars" (Kathy confirmed a few days ago that there shouldn't be any hyphen).